src/sentry/static/sentry/app/views/settings/team/teamMembers.jsx

```diff
@@ -5,8 +5,10 @@ import styled from 'react-emotion';
 import ApiMixin from '../../../mixins/apiMixin';
 import UserBadge from '../../../components/userBadge';
 import Button from '../../../components/buttons/button';
+import DropdownAutoComplete from '../../../components/dropdownAutoComplete';
+import DropdownButton from '../../../components/dropdownButton';
 import IndicatorStore from '../../../stores/indicatorStore';
-import {leaveTeam} from '../../../actionCreators/teams';
+import {joinTeam, leaveTeam} from '../../../actionCreators/teams';
```

Comment:
there's a memberListStore you may want to pull from and update in this component

Reply:
ok i will look into that, but probably not for this pr just in the interest of getting something usable out/letting chrissy start making it look nicer